src/components/StyleGuide/Filtering/PageCompositionElements/FilterBar.mdx

```diff
@@ -0,0 +1,232 @@
+
+  <Box>
+    - The `filter bar` should be placed above the area of the page that is pertinent
+      to the filters.
+    - The filters that compose the `filter bar` should render in one row for desktop
+      viewports (approx. five filter facets)
+      - For specific use cases that need more than five filter facets, the filters
+        should render on a maximum of two rows for desktop viewports
+    - Filters should be contextual and only apply to certain sections of the main
+      content area (such as the items table of the Research Catalog bib page)
+    - Using the `MultiSelect` component:
+      - `MultiSelect` components should be configured so that only one `MultiSelect`
+        component can be opened at any given time. (i.e. When a `MultiSelect`
+        component is opened, all other `MultiSelect` components will close.)
+        - NOTE: This configuration requires users to make selections in only one
+          opened dropdown at a time, making it slightly more difficult to quickly
+          make selections across multiple facets.
+      - `MultiSelect` components should be configured to use the `"scroll"` overflow
+        functionality
+  </Box>
+  <Image
+    alt="Example of row filter bar"
+    border="1px solid var(--nypl-colors-ui-border-default)"
+    caption="Example of row filter bar"
+    src={imageFilteringExampleRow}
+    {...exampleWrapperStyles}
+  />
+</Box>
+
+### Mobile
+
+<Box display="flex" flexDirection={{base: "column", md: "row"}} gap="l">
+
+  <Box>
+    For mobile viewports, the `FilterBarPopup` component should be used to render
+    the filter bar element. The `filter bar` element will not be visible when the
+    page first loads and users will need to click a "Show filters" button to open
+    the `filter bar` in a modal overlay.
+
+    The filter input fields will be presented in a column layout.
+
+    Using the `MultiSelect` component:
+
+    - `MultiSelect` components should be configured to allow multiple `MultiSelect`
+    components to be opened at the same time
+
+  </Box>
+    <Image
+    alt="Example of popup filters on mobile"
+    border="1px solid var(--nypl-colors-ui-border-default)"
+    caption="Example of popup filters on mobile"
+    src={imageFilteringExamplePopup}
+    {...exampleWrapperStyles}
+  />
+</Box>
+
+## Filter option labeling
+
+The label text for filter options should include the number of items associated
+with each filter option. For example, `English (5)` or `Offsite (11)`. The
+number of associated items included in the label text should be updated as
+filter options are applied. This treatment is applicable for the labels of the
+`Checkbox`, `Radio`, and `Toggle` components, and the labels of the checkbox
+options within a `MultiSelect` component, as well as the options within a
+`Select` component.
+
+## Filter option status
+
+If the number of items associated with a filter option is zero (0), that filter
```

Comment:
This would only apply to live filtering, correct? Or else it would only apply to filter options that are known beforehand. 
Most RC filters are based on aggregations that are returned with search responses, and can't be known before.

Reply:
If the associated items count comes back with the search response, once the response comes in, consuming apps should be able to update the data used to populate the component with the new numbers and states.